generators/app/templates/docker-compose.yml

```diff
@@ -5,8 +5,8 @@ services:
     image: <%= imageName %><% if (environment === 'debug') { %>:debug<% } %>
     build:
       context: .
-      dockerfile: Dockerfile<% if (environment === 'debug') { %>.debug<% } %><% if (isWebProject && portNumber) { %>
-    ports:
+      dockerfile: Dockerfile<% if (environment === 'debug') { %>.debug<% } %><% if ((isWebProject && portNumber) || (environment === 'debug' && debugPortNumber)) { %>
```

Comment:
Good point

---
In reply to: [71241206](https://github.com/Microsoft/generator-docker/pull/103#discussion_r71241206) [](ancestors = 71241206,71240674)